### DISCUSSION TITLE
New tag display option

### DISCUSSION THREAD

**Discussion by quickfold** [freeplane/freeplane#2034] [Ideas]
I want to use tags but I find them very distracting when they are displayed next to a node or under it. However, the only other current option is to not show them at all. I would like an option where the tags are displayed somewhere in the interface when you hover over a node with tags, perhaps in the bottom status bar (just like there is an option to show node ID). That way they are not distracting but there is a way to see the tags on a node without changing the tag display setting every time.

**Comment by euu2021** [comment]
> perhaps in the bottom status bar (just like there is an option to show node ID)

Another idea is to do like the other elements (notes, attributes etc): when they are in "hidden" mode, there is a symbol in the node showing that the element exists in that node, but it's only shown (in a tooltip) when the user hovers over it.

**Comment by quickfold** [comment]
@dpolivaev , any comment on these ideas?

**Comment by dpolivaev** [comment]
Thank you for sharing your thoughts! While I appreciate the suggestions, I work on Freeplane voluntarily, and without seeing concrete examples—specific maps, tag usage, and workflows—it’s very challenging to craft solutions that truly address your needs. Abstract proposals and arguments often leave too much to interpretation, which not only hinders my ability to deliver effective features but also feels demotivating.

If you want this feature to be considered, please help by sharing detailed examples or screenshots that clearly show how it would benefit your workflow. Real-life context helps me prioritize and inspires me to invest my time where it makes the biggest impact.

I look forward to your input—it makes all the difference!

**Reply by quickfold** [reply]
I'll give some more context with pictures: I like a simple undistracting display to organize information:

![image](https://github.com/user-attachments/assets/b89f234b-2599-4f3b-be4a-f0f2b9595ee3)

I want to use tags to help me quickly filter nodes in different combinations. Thus, I don't want the extra information always visible all of the time. Further, it's not like tags are just a bit of extra info you can overlook. Tags are a distracting, visual nightmare for people like me who don't like bright colors:

![image](https://github.com/user-attachments/assets/4ecb4f8e-1468-40d7-b039-cf6f35c9de51)

If I use tags, they are so distracting that I can't use the map for organizing information.

I understand that your general response to requests for specific features is to ask for context, details, etc., and that makes sense. However, in this case, I gave a clear concrete example of an existing problem and gave reasons for my view in my first post. I also suggested a concrete solution myself and asked for your opinion of that solution. My proposal is to have an option for tags so they are not always visible next to nodes, but instead appear in the lower status bar next to the node ID when the arrow hovers over each node. Thanks for considering and responding to my suggestions.

**Reply by dpolivaev** [reply]
Getting more details was helpful even in this very limited scope.
Do you think that using fix color for all  tags would help? You would be able to configure it in the styles.

**Reply by dpolivaev** [reply]
Showing a status icon  for nodes with tags and listing tags in a node tooltip is also doable.

**Reply by quickfold** [reply]
I do think that allowing the user to fix one color for all tags in styles would help. I tried changing the color for all tags to white one-by-one, and it was better. However, the tags still take up a lot of space in the map. I think either a status icon or node tooltip would be a better long-term solution. Thanks.

**Reply by euu2021** [reply]
> I do think that allowing the user to fix one color for all tags in styles would help.

I see that it exists in preview 1.12.9_06
